packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -326,6 +341,37 @@ function useId(): string {
   return id;
 }
 
+function useMemoCache(size: number): Array<any> {
+  const hook = nextHook();
+  let memoCache: MemoCache;
+  if (
+    hook !== null &&
+    hook.updateQueue !== null &&
+    hook.updateQueue.memoCache != null
+  ) {
+    memoCache = hook.updateQueue.memoCache;
+  } else {
+    memoCache = {
+      data: [],
+      index: 0,
+    };
+  }
+
+  let data = memoCache.data[memoCache.index];
+  if (data === undefined) {
+    data = new Array(size);
+    for (let i = 0; i < size; i++) {
+      data[i] = REACT_MEMO_CACHE_SENTINEL;
+    }
```

Comment:
nit: can this be simplified with `Array.fill`?

Reply:
I really am Grateful as I First started working on Chrome and afterwards Safari as I had been suffering alot with my "almost 1400 GB" Databases by the ebd of 2022 which took me in hell as no Device was taking and updating my databases properly. At the end of the day Exchange used to block my Devices either or may be my Internal Storages couldnt bear this load. 
Anyhow, besides using almost 10000 GB storages Online I had been suffering so far.
Love you Azure, Microsoft, GitHub, Google1, Apple and Yahoo!
Atlassian and Slack alongwith kdanCloud did a little job as well.

Move on please.